### PR TITLE
Themed Nav & Navbar components

### PR DESCRIPTION
**Pull request by kyleknighted**
## Description

- Themed `Nav`
- Themed `Navbar`

## Screenshots (if appropriate):
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: graylog2-web-interface/src/components/graylog/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+      > .open > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${darken(0.15, teinte.tertiary.uno)};
+          background-color: ${darken(0.065, teinte.secondary.due)};
+        }
+      }
+
+      @media (max-width: @grid-float-breakpoint-max) {
```
Comment (linuspahl) [inline graylog2-web-interface/src/components/graylog/Navbar.jsx]: I wish that would work, but as far as I know, this needs to be a fix width.

File: graylog2-web-interface/src/components/graylog/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+      > .open > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${teinte.primary.due};
+          background-color: ${lighten(0.20, teinte.primary.tre)};
+        }
+      }
+
+      @media (max-width: 767px) {
```
Comment (linuspahl) [inline graylog2-web-interface/src/components/graylog/Navbar.jsx]: I think this is the right time to store the breakpoints in e.g. the theme as well. What do you think? We would still need to unify them with the bootstrap variables, but it's an improvement, compared to hard coded values in the styles.

File: graylog2-web-interface/src/components/graylog/Navbar.jsx (hunk 1 above)
Comment (kyleknighted) [reply graylog2-web-interface/src/components/graylog/Navbar.jsx]: It was just a missed variable replacement, thanks for spotting it

File: graylog2-web-interface/src/components/graylog/Navbar.jsx (hunk 2 above)
Comment (kyleknighted) [reply graylog2-web-interface/src/components/graylog/Navbar.jsx]: Sounds good

File: graylog2-web-interface/src/components/graylog/Nav.jsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  }
+
+  &.nav-tabs.nav-justified {
+    > .active > a,
+    > .active > a:hover,
+    > .active > a:focus {
+      border-color: ${teinte.tertiary.quattro};
+    }
+
+    @media (min-width: 768px) {
```
Comment (linuspahl) [inline graylog2-web-interface/src/components/graylog/Nav.jsx]: You add the them breakpoint here as well. I really appreciate this implementation!

File: graylog2-web-interface/src/theme/breakpoints.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  sm: '768px',
+  md: '992px',
+  lg: '1200px',
+};
+
+const max = {};
+
+Object.keys(min).forEach((bp) => {
+  max[bp] = min[bp] - 1;
+});
```
Comment (linuspahl) [inline graylog2-web-interface/src/theme/breakpoints.js]: Looks like this will set every value in max to NaN

**Pull request merged**